clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -146,6 +148,8 @@ public VeniceChangelogConsumerImpl(
       this.userEventChunkingAdapter = GenericChunkingAdapter.INSTANCE;
       this.storeDeserializerCache = new AvroStoreDeserializerCache<>(storeRepository, storeName, true);
     }
+
+    avrostoreDeserializerCache = new AvroStoreDeserializerCache(storeRepository, storeName, true);
```

Comment:
looks good to me, just a nit: you could move this line up, and line 149 can reuse the same cache object for generic client?

Reply:
ugh!  Great point.

Reply:
fixed.

Reply:
Uh I don't think your PR will pass the unit test coverage. As I think last run should already failed on this https://github.com/linkedin/venice/actions/runs/7704830781

Reply:
on it.  This should pass